layers/ebclfsa/conf/cmake/ebclfsaPresets.json

```diff
@@ -15,7 +15,7 @@
             "toolchainFile": "/build/cmake/toolchain-ebclfsa-aarch64.cmake",
             "environment": {
                 "BUILDDIR": "/build/results/apps/$env{APP_NAME}/${presetName}/build",
-                "REALGCC": "aarch64-linux-gnu-gcc"
+                "REALGCC": "clang"
```

Comment:
As far as I can see, this is unused and can be removed. It was used by musl's gcc clang wrapper